sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryItemSerialization.java

```diff
@@ -58,53 +96,58 @@ private static byte[] decode(byte[] rawBytes) {
         }
     }
 
-    public static List<ByteBuffer> serialize(List<TelemetryItem> telemetryItems) {
-        try {
-            if (logger.canLogAtLevel(LogLevel.VERBOSE)) {
-                StringWriter debug = new StringWriter();
-                try (JsonGenerator jg = mapper.createGenerator(debug)) {
-                    writeTelemetryItems(jg, telemetryItems, mapper);
-                }
-                logger.verbose("sending telemetry to ingestion service:{}{}", System.lineSeparator(), debug);
+    // split the byte array by newline character
+    public static List<byte[]> splitBytesByNewline(byte[] inputBytes) {
```

Comment:
can we move this method to `LocalStorageTelemetryPipelineListener` since only called from there?